packages/components/collapsible-panel/src/collapsible-panel.spec.js

```diff
@@ -123,6 +124,38 @@ describe('when onToggle is provided without isClosed', () => {
   });
 });
 
+describe('condensed and when header prop has <CollapsiblePanelHeader>', () => {
```

Comment:
**nit**: perhaps we can use a test title like this:
`when using CollapsiblePanelHeader component as header`

and the test could be:
`should render an H4 title when condensed`
`should render an H2 title when not condensed`

Reply:
Thanks @CarlosCortizasCT! Makes more sense now. Updated here - [ba42b83](https://github.com/commercetools/ui-kit/pull/2548/commits/ba42b835ce0d135fd9401d3154ebdda2ea19eade).